package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Primer react components",
   "main": "dist/index.umd.js",
   "module": "dist/index.esm.js",
-  "typings": "index.d.ts",
+  "typings": "@types/primer/index.d.ts",
```

Comment:
Not sure how the `@types` packages are normally structured, but should this be `@types/primer/components` or `@types/primer-components`? There may be more libraries under the primer org in the future that we'll want to type

Reply:
Makes sense, should be updated to @types/primer-components/index.d.ts